test/fixtures/header.js

```diff
@@ -0,0 +1,5 @@
+module.exports = {
```

Comment:
Create a `components` folder inside `test`. Speaking of, `tests` instead of `test`.